README.md

```diff
@@ -1,116 +1,104 @@
-# Trapperkeeper-authorization
+# Trapperkeeper Authorization Service
 
 [![Build Status](https://travis-ci.org/puppetlabs/trapperkeeper-authorization.svg?branch=master)](https://travis-ci.org/puppetlabs/trapperkeeper-authorization)
 
-This clojure project is an authorization service for PuppetLabs Trapperkeeper.
-It aims to port Puppet's `auth.conf` feature to clojure, along with a different
-way to express authorization rules.
-
-The core of this service is a configuration format to express authorization
-rules which govern the REST API by parsing authentication information out of
-the incoming request, then matching the request against the configured rules.
-
-# Credits
-
-The original work for this library, service, and the original REST authconfig
-work in Ruby Puppet were all contributed by [Brice
-Figureau](https://github.com/masterzen).  This project has been graciously
-transferred to Puppet Labs for further development and maintenance as it
-becomes a critical part of the [Puppet
-Server](https://github.com/puppetlabs/puppet-server) security model as
-authconfig became a critical part of Puppet's security model.
+This project provides an authorization service for use with the
+[trapperkeeper service framework](https://github.com/puppetlabs/trapperkeeper).
+It aims to port Puppet's
+[`auth.conf`](https://docs.puppetlabs.com/puppet/latest/reference/config_file_auth.html)
+feature to Clojure and the trapperkeeper framework, with a different way to 
+express authorization rules.
 
 ## Installation
 
-Add the following to your _leiningen_ `project.clj`:
+To use this service in your trapperkeeper application, simply add this project
+as a dependency in your leiningen project file:
 
 [![Clojars Project](http://clojars.org/puppetlabs/trapperkeeper-authorization/latest-version.svg)](http://clojars.org/puppetlabs/trapperkeeper-authorization)
 
+Then add the authorization service to your
+[`bootstrap.cfg`](https://github.com/puppetlabs/trapperkeeper/wiki/Bootstrapping)
+file, via:
+
+    puppetlabs.trapperkeeper.services.authorization.authorization-service/authorization-service
+
+The authorization service provides an implementation of the
+ `:AuthorizationService` interface.
+
+The authorization service is configured via the
+[trapperkeeper configuration service](https://github.com/puppetlabs/trapperkeeper/wiki/Built-in-Configuration-Service);
+so, you can control the authorization logic by adding an `authorization` section
+to one of your Trapperkeeper configuration files, and setting various 
+properties therein.  For more info, see
+[Configuring the Authorization Service](doc/authorization-config.md).
```

Comment:
I almost feel like we should swap the information in this README and this config document.  Aren't people more interested in the `auth.conf` file rather than how the TK service works?  This documentation is very valuable, but shouldn't we target this README at users rather than devs?